src/index.spec.js

```diff
@@ -580,15 +580,17 @@ describe('Dropzone', () => {
         dataTransfer: { files: images.concat(files) }
       })
       const rejected = dropSpy.firstCall.args[1]
-      expect(rejected.length).toEqual(2)
+      expect(rejected.length).toBeGreaterThan(1)
     })
 
     it('should allow single files to be dropped if multiple is false', () => {
-      const dropzone = mount(<Dropzone accept="image/*" onDrop={dropSpy} multiple={false} />)
+      const dropzone = mount(
+        <Dropzone accept="image/*" onDrop={dropAcceptedSpy} multiple={false} />
+      )
 
-      dropzone.simulate('drop', { dataTransfer: { files: [images[0]] } })
-      const [accepted, rejected] = dropSpy.firstCall.args
-      expect(accepted.length).toEqual(1)
+      dropzone.simulate('drop', { dataTransfer: { files: images[0].accept } })
```

Comment:
What is `images[0].accept`? `files` must be an array, right? Why did you change this line?

Reply:
sry, gonna change it back with the next commit.